koji/collection-mash-split.py

```diff
@@ -399,7 +400,7 @@ def main():
 
         mashes = [MashConfig(collection, version, "rhel7-dist", "rhel7", "RHEL/7", extras)]
 
-        if version not in ['2.0']:
+        if LooseVersion(version) > LooseVersion('2.0'):
```

Comment:
Is this also true for `nightly`? Given 2.0 is now EOL, should we first submit a PR that cleans up 2.0?

Reply:
I think that's kinda chicken/egg, as I'm working now on the cleanup PRs.  So i'll just take it into account, one way or another.

Reply:
My logic is always to made the changes in an order that results in the smallest diffs. In this case that means first cleaning up all the 2.0 logic and then introducing `LooseVersion`. I adopted this because it makes tracking regressions easier and now apply it to all my patches.